src/Core/Framework/App/Command/ChangeShopIdCommand.php

```diff
@@ -13,30 +13,35 @@
 use Symfony\Component\Console\Output\OutputInterface;
 
 /**
- * @internal only for use by the app-system
+ * @internal
+ *
+ * @deprecated tag:v6.8.0 - The command's name will change to `app:shop-id:change`
  */
 #[AsCommand(
     name: 'app:url-change:resolve',
-    description: 'Resolves app url changes',
+    description: 'Change the shop ID by choosing a resolution strategy',
+    aliases: ['app:shop-id:change'],
 )]
 #[Package('framework')]
-class ResolveAppUrlChangeCommand extends Command
+class ChangeShopIdCommand extends Command
 {
-    public function __construct(private readonly Resolver $appUrlChangeResolver)
+    public function __construct(private readonly Resolver $shopIdChangeResolver)
     {
         parent::__construct();
     }
 
+    // @phpstan-ignore-next-line shopware.deprecatedClass - Annotation can be removed with tag:v6.8.0 when the deprecation is removed
```

Comment:
```suggestion
    // @phpstan-ignore shopware.deprecatedClass (Annotation can be removed with tag:v6.8.0 when the deprecation is removed)
```

ignore-next-line will ignore every PHPStan issue and not just the specific one.